web/api/mocks/service/StorageService.js

```diff
@@ -13,7 +13,9 @@ let collections = {
   'flows': require('../data/flows.json'),
   'steps': require('../data/steps.json'),
   'collections': require('../data/collections.json'),
-  'jobs': []  // generated on the fly when /flow/{flow id}/run triggered
+  'jobs': [],  // generated on the fly when /flow/{flow id}/run triggered
+  'job-detail': require('../data/job.json'),
```

Comment:
Seeding "jobs" with the Array in jobs.json is what should happen here and all additions/deletions/gets/updates should happen against the "job" collection.